### PR TITLE
Update UG, Add Command, Command Summary, Constraints

### PR DESCRIPTION
**Pull request by alyssa-savier**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -93,15 +93,21 @@ constraints can be found in the [parameter constraints.](#parameter-constraints-
 
 Adds a new guest or staff and their contact details into PH.
 
-Format: `add n/NAME pn/PASSPORT_NUMBER [p/PHONE_NUMBER] [r/ROOM_NUMBER]`
+Format: `add n/<NAME> pn/<PASSPORT_NUMBER> [p/<PHONE_NUMBER>] [r/<ROOM_NUMBER>]`/ `add n/<NAME> sid/<STAFF_ID> [p/<PHONE_NUMBER>] [r/<ROOM_NUMBER>]`
```
Comment (NicolasCwy) [inline docs/UserGuide.md:96]: Check out Jeremy's PR #27 for the format!

**Review by NicolasCwy — CHANGES_REQUESTED** [review]
Small formatting issue!

**Review by NicolasCwy — APPROVED** [review]

**Review by NicolasCwy — APPROVED** [review]
LGTM, I will change the format on my side

**Review by bingcheng45 — APPROVED** [review]
LGTM

**Pull request merged**